site/content/en/blog/releases/v1.24.0.md

```diff
@@ -13,6 +13,12 @@ description: Cloud Provider Azure v1.24.0
 - Feat: add DiskEncryptionType in CreateDisk ([#1646](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1646), [@andyzhangx](https://github.com/andyzhangx))
 - Update dependencies to 1.24 ([#1655](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1655), [@MartinForReal](https://github.com/MartinForReal))
 
+## Actions Required
+
+From cloud provider Azure v1.24.0 version, the LoadBalancer typed services with appProtocol HTTP/HTTPS would switch to HTTP/HTTPS for health probe protocol (while TCP is used before v1.24.0). And `/` would be used as the default health probe request path. If your service doesn't respond 200 for `/`, please ensure set service annotation `service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path` with correct request path to avoid service broken.
```

Comment:
IMO, annotation service.beta.kubernetes.io/port_{port}_health-probe_request-path would be recommended .

Reply:
added the annotation as well.

Reply:
created a new PR https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1690 due to branch policies.